### PR TITLE
Migrate khan/kmath into monorepo

### PR DESCRIPTION
**Pull request by jeremywiebe**
## Summary:

It's Friday, so I thought I'd see how hard it was to migrate one of our sub-repos into this monorepo. Turns out (for kmath, at least) it wasn't too hard. 

This PR _doesn't_ yet publish as we haven't set that up for this repo yet.
This PR also doesn't attempt to retain the history from the [khan/kmath](https://github.com/khan/kmath) repo (I'm not sure that's desirable)

Issue: LP-11430

## Test plan:

`yarn test` passes
`yarn build` works
`yarn --cwd packages/kmath pack` results in a bundle in `packages/kmath`

### CODE REVIEW

File: packages/kmath/.babelrc.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/**
+ * HACK(somewhatabstract): Due to https://github.com/facebook/jest/issues/11741,
+ * we need to have this file, or updating inline snapshots can fail rather
+ * cryptically.
+ *
+ * We should remove this when jest is fixed.
+ */
+module.exports = require("../../config/build/babel.config");
```
Comment (jeremywiebe) [inline packages/kmath/.babelrc.js:8]: WonderStuff does this so I'm assuming we need to.

**Review by kevinbarabash — COMMENTED** [review]
Thanks for migrating this package.  It'll be nice to have all of the code and tests together in one place.  I think this needs a couple of tweaks to the package.json.  Thanks for checking that the package builds correctly in the test plan.  Could you also update perseus/src/index.js to import `kmath` and make use of one of its functions?  This will probably require tweaks to the module mapping in the flow and jest configuration.

**Comment by jeremywiebe** [comment]
@kevinbarabash you reviewed this PR before I'd taken it out of draft. Once I did, it requires another review. Sorry.

File: packages/kmath/package.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+    "publishConfig": {
+        "access": "public"
+    },
+    "name": "@khanacademy/kmath",
+    "version": "0.0.3",
+    "description": "Khan Academy's Javascript Numeric Math Utilities",
+    "module": "dist/es/index.browser.js",
+    "main": "dist/index.browser.js",
```
Comment (kevinbarabash) [inline packages/kmath/package.json:9]: This should be updated to:
```
"module": "dist/es/index.js",
"main": "dist/index.js",
```
after merging in `main`.

**Review by kevinbarabash — APPROVED** [review]
LGTM.  Thanks for tweak the deps.  You'll have to modify a `"main"` and `"module"` fields in package.json after merging in `main`.

**Pull request merged**